### ISSUE TITLE
a11y: keyboard navigation is not selecting the RKE1/RKE2 toggle

### ISSUE THREAD

**Issue by IsaSih** [rancher/dashboard#13296]
**Setup**
- Rancher version: 
Rancher | v2.11-a1bda936c809e0394520c9d40d2998529223401e-head
Dashboard | master 8795891
Rancher CLI | v2.11.0-rc.1

**Describe the bug**
In the cluster provisioning page, the keyboard navigation is not selecting the RKE1/RKE2 toggle

**To Reproduce**
1- From the cluster provisioning page,hit Create
2- use the keyboard navigation to select the RKE1/RKE2 toggle

**Result**
the keyboard navigation is not selecting the RKE1/RKE2 toggle

**Expected Result**
The keyboard navigation should be able to select the RKE1/RKE2 toggle and allow the user to switch between RKE1 and RKE2 with enter/return/space key

**Screenshots**

https://github.com/user-attachments/assets/a82a0376-a9c1-47df-b189-e49c33189bbb

**Comment by aalves08** [comment]
Related to https://github.com/rancher/dashboard/issues/12817. Will be fixed with https://github.com/rancher/dashboard/pull/13259 as per https://github.com/rancher/dashboard/pull/13259#issuecomment-2637823806

**Comment by IsaSih** [comment]
Tests pass in 
Rancher | v2.11-3d729db8a848fb2523bbc2f3f21aa5367dcdcc69-head
Dashboard | master df18acc
Rancher CLI | v2.11.0-rc.1
